js/client.js

```diff
@@ -37,8 +37,52 @@ function getThreadClient() {
   return currentThreadClient;
 }
 
+const unsolicitedEvents = [
```

Comment:
Why are you copying unsolicitedEvents here? It looks like you are only using it to log the protocol, but this isn't the place to do it. If you set `require("devtools/shared/DevToolsUtils").dumpn.wantLogging = true` it should enable verbose logging which should log the protocol. If not, we just need to fix that in `ff-devtools-libs` because that's how it works in gecko-dev. (that setting is also flipped with a pref)